### DISCUSSION TITLE
input buffer overflow, can't enlarge buffer because scanner uses REJECT

### DISCUSSION THREAD

**Discussion by janboeye** [YosysHQ/yosys#4658] [Q&A] [answered]
hi, 
When write_verilog tmp.v after synth :fine and then read_verilog tmp.v，“input buffer overflow, can't enlarge buffer because scanner uses REJECT” is reported.

tmp.v size is about 300MB.

what's the right command to write out intermedia result?

Thanks a lot!

**Comment by nakengelhardt** [comment, marked as answer]
To directly write the Yosys internal representation to disk, you can use `write_rtlil` and then read it back in with `read_rtlil`.
